src/SFML/Window/Win32/WindowImplWin32.cpp

```diff
@@ -1011,6 +1123,73 @@ void WindowImplWin32::processEvent(UINT message, WPARAM wParam, LPARAM lParam)
 
             break;
         }
+        case WM_TOUCH:
+        {
+            // Get the number of events
+            std::int16_t num = LOWORD(wParam);
+            
+            // Reserve memory
+            PTOUCHINPUT events = new TOUCHINPUT[num];
```

Comment:
Could we instead use a vector here or at least a unique_ptr to a C-style array so that we don't have to manually `delete` this later?

Reply:
One could probably just set a reasonably large array size an avoid reallocating completely.